_posts/2019-01-29-yocto-meta-rte-migrate-to-github.md

```diff
@@ -0,0 +1,90 @@
+﻿---
+post_title: Yocto meta-rte migrate to GitHub
+author: Marta Szelecka
+layout: post
+published: false
+post_date:
+
+tags:
+    - Yocto
+    - meta-rte
+    - linux
+    - rte
+categories:
+    - OS Dev
+---
+
+## Yocto meta-rte migrate to GitHub
+
+We are happy to announce that our 3mdeb’s Yocto `meta-rte` is now available on our GitHub.

```

Comment:
@MartaSzel here should be linked our GitHub url - at this moment this should be ok:  ( \[GitHub\]\(link-TBD\) )
After `meta-rte` migration take place, you should update link

Reply:
@MartaSzel https://github.com/3mdeb/meta-rte